### DISCUSSION TITLE
Nuxt server components + slot is broken?

### DISCUSSION THREAD

**Discussion by moebiusmania** [nuxt/nuxt#21511] [Questions] [answered]
hello,
I'm toying a bit with Server Components in Nuxt (https://nuxt.com/docs/guide/directory-structure/components#server-components) and it looks like that `<slot />` are broken in `.server` components file, it renders just part of the combined result. 

I'm not finding any information about it right now, so before opening an issue I want to understand if that's its the current situation  (since it's an experimental feature) and someone else is aware about it.

Any answer is appreciated, thanks.

EDIT: I'm talking about slotting `.server` components within `.server` components, not mixing the `.server` and `.client` ones

**Comment by moebiusmania** [comment, marked as answer]
ok it looks like I haven't carefully watched the video embedded in the page as torwards the end it explicitly says that Server Components aren't supporting slot at the moment.

**Reply by texastoland** [reply]
I haven't tried it myself but I was think they were supposed to be implemented in #19765 👀

**Reply by moebiusmania** [reply]
I've tried a few days back on this toy project https://github.com/moebiusmania/atmmi-info-ui and things inside the `<slot />` doesn't render at all 😞

**Reply by texastoland** [reply]
Open an issue?
